actions.go

```diff
@@ -1,4 +1,4 @@
-package main
+package ejsonkms
```

Comment:
I _think_ these files should now live in an `ejsonkms` directory. What do you think?

Reply:
I think it's fine for these to be in the root since they're the core of the package. ejson and ejson2env also follow this approach.